docker-compose.yml

```diff
@@ -26,14 +24,14 @@ services:
     links:
       - ipfs
       - postgres
-      {{#if development}}- ganache{{/if}}
+      - ganache
     environment:
       postgres_host: postgres:5432
       postgres_user: postgres
-      postgres_pass: '{{postgresPassword}}'
+      postgres_pass: 'letmein'
```

Comment:
why this password is here (public)?

Reply:
its just for local tests, once we have a production deployment we'll have it secret.

Reply:
so for production we will have different docker-compose.yml  ?
if yes , please rename the file to dev-docker-compose.yml

Reply:
lets worry about this when we deploy production?

Reply:
we already want to use testnet ..
so lets call this dev-docker-compose.yml

Reply:
lets decide that the default `docker-compose.yml` is the testing setup, and others will be `docker-compose.kovan.yml`, `docker-compose.mainnet.yml`. Agree?
Otherwise it will make commands ugly having to specify `-f dev-docker-compose.yml` all the time while developing.

Reply:
ok